libs/vectorscan/Makefile

```diff
@@ -24,8 +24,6 @@ PKG_BUILD_DEPENDS:=ragel/host python3/host boost/host
 include $(INCLUDE_DIR)/package.mk
 include $(INCLUDE_DIR)/cmake.mk
 
-DEPENDS_COMMON:=@(x86_64||aarch64)
-
 # With at least version 5.4.12, Neon/ASIMD is required for Arm support
 ifeq ($(CONFIG_CPU_NEON),)
   PKG_BUILD_FLAGS := skip
```

Comment:
Hey @graysky2 , this is still not right.

If you enable vectorscan for 32-bit arm, then GitHub CI/CD for [arm_cortex-a15_neon-vfpv4](https://github.com/openwrt/packages/actions/runs/17986754927/job/51166797601#step:7:29983) tries to compile it. So, this needs a little bit work. :) Which you can do.